README.md

```diff
@@ -92,11 +92,11 @@ To access the machine learning service programmatically, you need to copy in you
 * Open the `Service Credentials` tab and view the credentials.
 * Copy your credentials and replace the `**URL**`, `**USERNAME**`, `**PASSWORD**` and `**INSTANCE_ID**` placeholders in the notebook and run the cell.
 
-## 6. Deploy the model to the Cloud and run the product recommendation PixieApp
+### 6. Deploy the model to the Cloud and run the product recommendation PixieApp
```

Comment:
Why the demotion to H4 from H3?

Reply:
Good catch - I'm not sure what I was thinking. I'll revert.